core/vm/evm.go

```diff
@@ -199,12 +207,24 @@ func (evm *EVM) Call(caller ContractRef, addr common.Address, input []byte, gas
 	snapshot := evm.StateDB.Snapshot()
 	p, isPrecompile := evm.precompile(addr)
 
+	var creation bool
```

Comment:
IF the `addr` is prefunded with balance, and this happens later, then the check `if !evm.StateDB.Exist(addr)` will be false, and the `created = true` will not be set.

Reply:
After some investigation: this `Call` method _never_ handles initcode (?)